src/storage/object/delete.js

```diff
@@ -71,5 +71,5 @@ export default async function deleteObjects(env, daCtx) {
     }
   } while (ContinuationToken);
 
-  return { body: '', status: 204 };
+  return { body: null, status: 204 };
```

Comment:
Note that this change was because of the following message that I observed in the console:

```
✘ [ERROR] Constructing a Response with a null body status (204) and a non-null, zero-length body. This is technically incorrect, and we recommend you update your code to explicitly pass in a `null` body, e.g. `new Response(null, { status: 204, ... })`. (We continue to allow the zero-length body behavior because it was previously the only way to construct a Response with a null body status. This behavior may change in the future.)
```